llm-service/README.md

```diff
@@ -408,8 +433,9 @@ oc describe inferenceservice llama-3-2-3b-instruct
 2. **GPU Resource Issues**:
    - Verify GPU nodes are available
    - Check GPU resource limits
-   - Ensure NVIDIA device plugin is running
-   - Validate GPU toleration settings
+   - For NVIDIA GPUs: Ensure NVIDIA device plugin is running
+   - For AMD GPUs: Ensure AMD ROCm device plugin is running
```

Comment:
Any idea how we can ensure that the AMD ROCm device plugin is running before the software is installed ? I am guessing that Openshift will take care of this somehow ?